app/bundles/CampaignBundle/Entity/CampaignRepository.php

```diff
@@ -605,8 +605,7 @@ public function getCampaignLeadIds($campaignId, $start = 0, $limit = false, $pen
         }
 
         if (!empty($limit)) {
-            $q->setFirstResult($start)
```

Comment:
This makes sense for when $pendingOnly = true. Can you wrap this in `if (!$pendingOnly && $start)` instead of removing it?